packages/odie-client/src/constants.ts

```diff
@@ -46,6 +46,14 @@ export const ODIE_TRANSFER_MESSAGE: Message[] = [
 	},
 ];
 
+export const ODIE_THIRD_PARTY_MESSAGE = `${ __(
```

Comment:
Since we can't detect if someone's disabled 3rd party cookies easily, and the error connecting to ZenDesk may have nothing to do with cookies – could we make it a little more ambiguous? i.e. "This may be because 3rd party cookies are disabled in your browser."